db/migrate/20250602194448_update_local_identifiers_cached_numeric.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateLocalIdentifiersCachedNumeric < ActiveRecord::Migration[7.2]
+  def change
+    Identifier::Local.find_each do |i|
```

Comment:
@kleintom I think we can be more efficient here?  Does this only apply to a subclass of identifiers, or identifier with a particular namespace?  I.e. `Idenfiier...join(:namespace).where(namespace: { is_virtual: true})...`